README.md

```diff
@@ -1,4 +1,4 @@
-# PDF.js [![Build Status](https://github.com/mozilla/pdf.js/workflows/CI/badge.svg?branch=master)](https://github.com/mozilla/pdf.js/actions?query=workflow%3ACI+branch%3Amaster)
+# PDF.js [![CI](https://github.com/mozilla/pdf.js/actions/workflows/ci.yml/badge.svg)](https://github.com/mozilla/pdf.js/actions/workflows/ci.yml)
```

Comment:
If you look at the old link you'll notice that we (purposely) targeted the `master` branch to avoid showing unrelated build-state.
Hence the following would be a more appropriate link: https://github.com/mozilla/pdf.js/actions/workflows/ci.yml?query=branch%3Amaster